esp32c6-hal/examples/lp_core_basic.rs

```diff
@@ -61,6 +131,12 @@ fn main() -> ! {
     wdt0.disable();
     wdt1.disable();
 
+    let io = IO::new(peripherals.GPIO, peripherals.IO_MUX);
+
+    // configure GPIO 1 as LP output pin
+    let mut lp_pin = io.pins.gpio1.into_low_power();
+    lp_pin.output_enable(true);
```

Comment:
This is merged already so I missed the boat, but I don't think it makes a ton of sense having different APIs for the HP and LP GPIO.

Reply:
It's the first step and there are plenty of changes upcoming - so many chances to revise the API. 
For now, the idea is to have it look like esp-idf to make it easy to compare until we know everything works fine.